mavros/src/plugins/actuator_control.cpp

```diff
@@ -35,25 +35,45 @@ class ActuatorControlPlugin : public plugin::PluginBase {
 	{
 		PluginBase::initialize(uas_);
 
+		actuator_control_target_pub = nh.advertise<mavros_msgs::ActuatorControl>("actuator_control", 10);
 		actuator_control_sub = nh.subscribe("actuator_control", 10, &ActuatorControlPlugin::actuator_control_cb, this);
```

Comment:
Lets be similat ro setpoint_raw: actuator_control - sub; target_actuator_control - pub